File: OneCodeTeam/Claim-aware-WCF-WebServicevs10-17865/[VB]-Claim-aware WCF WebService[vs10] (VBSharePointCallClaimsAwareWCF)/VB/VBSharePointCallClaimsAwareWCF/Service/CustomServiceInstance.vb
'*************************** Module Header ******************************\
' Module Name:    CustomServiceInstance.vb
' Project:        VBSharePointCallClaimsAwareWCF
' Copyright (c) Microsoft Corporation
'
' This is the CustomServiceInstance Class
' 
' This source is subject to the Microsoft Public License.
' See http://www.microsoft.com/en-us/openness/licenses.aspx#MPL.
' All other rights reserved.
' 
' THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
' EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
' WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
'\****************************************************************************


Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports Microsoft.SharePoint.Administration
Imports CustomService

Namespace CustomService
    Public Class CustomServiceInstance
        Inherits SPIisWebServiceInstance
        Public Sub New()
        End Sub

        Public Sub New(ByVal server As SPServer, ByVal service As CustomService)
            MyBase.New(server, service)
        End Sub

        Public Overrides ReadOnly Property TypeName() As String
            Get
                Return "Custom Service"
            End Get
        End Property
    End Class
End Namespace

File: OneCodeTeam/Claim-aware-WCF-WebServicevs10-17866/[C#]-Claim-aware WCF WebService[vs10] (CSSharePointCallClaimsAwareWCF)/C#/CSSharePointCallClaimsAwareWCF/Service/CustomServiceInstance.cs
/****************************** Module Header ******************************\
* Module Name:    CustomServiceInstance.cs
* Project:        CSSharePointCallClaimsAwareWCF
* Copyright (c) Microsoft Corporation
*
* This is the CustomServiceInstance Class
* 
* This source is subject to the Microsoft Public License.
* See http://www.microsoft.com/en-us/openness/licenses.aspx#MPL.
* All other rights reserved.
* 
* THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
* EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
* WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
\*****************************************************************************/

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Microsoft.SharePoint.Administration;
using CustomService;

namespace CustomService
{
    public class CustomServiceInstance : SPIisWebServiceInstance
    {
        public CustomServiceInstance() { }

        public CustomServiceInstance(SPServer server, CustomService service)
            : base(server, service) { }

        public override string TypeName
        {
            get { return "Custom Service"; }
        }
    }
}